packages/nx-plugin/src/plugin/types.ts

```diff
@@ -3,8 +3,10 @@ import { DynamicTargetOptions } from '../internal/types';
 
 export type CreateNodesOptions = DynamicTargetOptions;
 
+export type ProjectConfigWithName = Omit<ProjectConfiguration, 'name'> &
```

Comment:
I think is something we could create a utility type for. Can be implemented [like this](https://www.typescriptlang.org/play/?ts=5.3.3#code/PTAEDMCcHsFtQHIA8BQAXAngBwKagAowBWOAxmgMLQB24AlgOYCukAhmnTaALygDeKUKGqtYOAPwAuUAGc0kOtQYBuQaBjQ00uQqWqAvihQhQAExwKAbjlOhMudNjyFoJclVqMW7TtQDqdGgAFgiieLwBwQBKOACOTHSQNgA8Lm6UNPTMbBw0ADSgAOQiYoUAfEYmiqBMHAA2Mig4SFjQkGh2TqCRQTHxiSkAKgUA0qDNaDjUpjKgANY4GNDgoINlPARJaBzgGMkA8rCBycOgI+sAZKB9CUmmqXSkcyejZW9NLW0d9s5bO3trDZ8UAAbTG1QWSxWgwAutJBmCYaBDEA).

![image](https://github.com/user-attachments/assets/08d7ffcb-0cfd-451b-b07d-6789bb9be0bb)

Reply:
🔥